src/contract/dao/dao-app-approval.ts

```diff
@@ -1,1115 +1,1120 @@
 import { compileToUnit8Array } from "@/utility";
 
 const daoAppApproval = `#pragma version 6
```

Comment:
@Megha-Dev-19 The previous single tab indentation was correct. Please add single tab indentation.